static/app/views/replays/detail/layout/focusTabs.tsx

```diff
@@ -27,41 +29,44 @@ function getReplayTabs({
 
 type Props = {
   isVideoReplay: boolean;
-  className?: string;
 };
 
-function FocusTabs({className, isVideoReplay}: Props) {
+function FocusTabs({isVideoReplay}: Props) {
   const organization = useOrganization();
   const {pathname, query} = useLocation();
+  const navigate = useNavigate();
   const {getActiveTab, setActiveTab} = useActiveReplayTab({isVideoReplay});
   const activeTab = getActiveTab();
 
   return (
-    <ScrollableTabs className={className} underlined>
-      {Object.entries(getReplayTabs({isVideoReplay})).map(([tab, label]) =>
-        label ? (
-          <ListLink
-            data-test-id={`replay-details-${tab}-btn`}
-            key={tab}
-            isActive={() => tab === activeTab}
-            to={{pathname, query: {...query, t_main: tab}}}
-            onClick={e => {
-              e.preventDefault();
-              setActiveTab(tab);
-
-              trackAnalytics('replay.details-tab-changed', {
-                tab,
-                organization,
-                mobile: isVideoReplay,
-              });
-            }}
-          >
-            {label}
-          </ListLink>
-        ) : null
-      )}
-    </ScrollableTabs>
+    <TabContainer>
+      <Tabs
+        value={activeTab}
+        onChange={tab => {
+          setActiveTab(tab);
+          navigate({
+            pathname,
+            query: {...query, t_main: tab},
+          });
+          trackAnalytics('replay.details-tab-changed', {
+            tab,
+            organization,
+            mobile: isVideoReplay,
+          });
+        }}
+      >
+        <TabList>
+          {Object.entries(getReplayTabs({isVideoReplay})).map(([tab, label]) => (
+            <TabList.Item key={tab}>{label}</TabList.Item>
+          ))}
+        </TabList>
+      </Tabs>
+    </TabContainer>
```

Comment:
This is having some trouble when there's overflow because the window is narrow:

![SCR-20250324-ogph](https://github.com/user-attachments/assets/a06c3472-9ef2-4a5a-9a4f-2db4aa499b2a)